Dockerfile.ubi.cpu.ppc64le

```
@@ -0,0 +1,256 @@
+    && curl -sL https://ftp2.osuosl.org/pub/ppc64el/openblas/latest/Openblas_0.3.29_ppc64le.tar.gz | tar xvf - -C /usr/local \
+    && ln -sf $(command -v python${PYTHON_VERSION}) /usr/local/bin/python \
+    && ln -sf $(command -v pip${PYTHON_VERSION}) /usr/local/bin/pip
+ENV PATH=/root/.cargo/bin:$PATH
+ENV PKG_CONFIG_PATH=/usr/local/lib/pkgconfig/
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib64:/usr/local/lib:/usr/lib64:/usr/lib
+RUN --mount=type=cache,target=/root/.cache/pip \
+    --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,from=torch-builder,source=/pytorch/dist,target=/pytorch,rw \
+--mount=type=bind,from=torch-builder,source=/vision/dist,target=/torchvision,rw \
+    --mount=type=bind,from=torch-builder,source=/audio/dist,target=/torchaudio,rw \
+--mount=type=bind,from=arrow-builder,source=/arrow/python/dist,target=/pyarrow,rw \
+--mount=type=bind,from=cv-builder,source=/opencv-python/dist,target=/opencv,rw \
+    ls -ltr /pytorch && ls -ltr /torchvision && ls -ltr /torchaudio && ls -ltr /pyarrow && ls -ltr /opencv && \
+    source /opt/rh/gcc-toolset-13/enable && \
+    pip install /pytorch/torch*.whl && \
+    pip install /torchvision/torchvision*.whl && \
+    pip install /torchaudio/torchaudio*.whl && \
+    pip install /pyarrow/pyarrow*.whl && \
+    pip install /opencv/opencv*.whl
+
+WORKDIR /workspace
+
+RUN --mount=type=cache,target=/root/.cache/pip \
+    --mount=type=bind,src=.,dst=/workspace/vllm,rw \
+    cd vllm && \
+    source /opt/rh/gcc-toolset-13/enable && \
+    python use_existing_torch.py \
+    && ln -sf /usr/lib64/libatomic.so.1 /usr/lib64/libatomic.so && \
+    pip install -v cython setuptools_scm setuptools_rust packaging \
+    "numpy<2.0" pandas==2.0 pillow scikit_build_core scikit-build meson-python sentencepiece outlines-core \
+    && pip install -v -r requirements-build.txt \
+    && pip install -v . --no-build-isolation
+
+# setup non-root user for OpenShift
+RUN umask 002 && \
+    useradd --uid 2000 --gid 0 vllm && \
+    mkdir -p /home/vllm && \
+    chmod g+rwx /home/vllm /usr/src /workspace
+COPY LICENSE /licenses/vllm.md
+COPY examples/*.jinja /app/data/template/
+
+USER 2000
+
+WORKDIR /home/vllm
+
+ENTRYPOINT ["python", "-m", "vllm.entrypoints.openai.api_server"]
+
+#############################################################
+
+# VLLM TGIS ADAPTER STAGE
+
+FROM vllm-builder as vllm-grpc-adapter
+ARG VLLM_TARGET_DEVICE=cpu
+USER root
+
+# Vllm-tgis-adapter 0.6.2 has an issue 
+RUN --mount=type=cache,target=/root/.cache/pip \
+    --mount=type=cache,target=/root/.cache/uv \
+    source /opt/rh/gcc-toolset-13/enable && \
+    git clone -b 0.6.2 https://github.com/opendatahub-io/vllm-tgis-adapter.git && cd vllm-tgis-adapter && git cherry-pick --no-commit 178d4bafcf6cb2ff8d8088d56ecc167572c83329 \
+    && pip install build && \
+    HOME=/root GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 python -m build -w && \
+    pip install dist/*.whl && cd .. && rm -rf vllm-tgis-adapter
```

I'll be releasing vllm-tgis-adapter 0.6.3 with the given fix soon, so we can move to this:

```suggestion
    HOME=/root GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 uv pip install git+https://github.com/opendatahub-io/vllm-tgis-adapter@0.6.3
```